### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by karanshah-browserstack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,80 @@
...
+
+### **Integrate your test suite**
+This repository uses the BrowserStack SDK to run tests on BrowserStack. Follow the steps below to install the SDK in your test suite and run tests on BrowserStack:
+
+- Create sample browserstack.yml file with the browserstack related capabilities with your BrowserStack Username and Access Key and place it in your root folder.
+- Add nuget library BrowserStack.TestAdapter
+```sh
+dotnet add BrowserStack.TestAdapter
+```
+- Build project dotnet build
```
Comment (sriteja777) [inline README.md]:
```suggestion
- Build project `dotnet build`
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,80 @@
...
+
+--
+
+## Integration with other CSharp frameworks
+
+For other CSharp frameworks samples, refer to following repositories :
+
+- [Appium CSharp tests](https://github.com/browserstack/csharp-appium-app-browserstack)
```
Comment (sriteja777) [inline README.md:60]: Do we want to add nunit, specflow?

File: android/.config/dotnet-tools.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "isRoot": true,
+  "tools": {
+    "browserstack-sdk": {
+      "version": "1.16.3",
+      "commands": [
+        "browserstack-sdk"
+      ]
+    }
+  }
+}
```
Comment (sriteja777) [inline android/.config/dotnet-tools.json]: new line

File: android/BaseFixture.cs [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+        public AndroidDriver<AndroidElement> GetDriver(string platform, string profile)
+        {
+
+            // Create Desired Cappabilities for WebDriver
+            AppiumOptions appiumOptions = new AppiumOptions();
+
+            // Create RemoteWebDriver instance
+            driver = new AndroidDriver<AndroidElement>(
+                new Uri($"http://127.0.0.1:4723/wd/hub"), appiumOptions);
```
Comment (sriteja777) [inline android/BaseFixture.cs:23]: do we want to add source if missing

File: ios/.config/dotnet-tools.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  "isRoot": true,
+  "tools": {
+    "browserstack-sdk": {
+      "version": "1.16.3",
+      "commands": [
+        "browserstack-sdk"
+      ]
+    }
+  }
+}
```
Comment (sriteja777) [inline ios/.config/dotnet-tools.json]: new line

File: README.md (hunk 2 above)
Comment (karanshah-browserstack) [reply README.md:60]: Not doing as of now.

File: android/BaseFixture.cs (hunk 4 above)
Comment (karanshah-browserstack) [reply android/BaseFixture.cs:23]: Keeping consitent with existing xunit sample repos

**Review by khrithik — DISMISSED** [review]

**Review by khrithik — APPROVED** [review]

**Review by pranavj1001 — APPROVED** [review]

**Pull request merged**